scripts/rotateLatLong.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+import json
+
+with open("in.json", "r") as f:
+    data = json.loads(f.read())
+
+def rotateWorker(x):
+    if isinstance(x[0], list):
+        for el in x:
+            rotateWorker(el)
+    else:
+        first = x[0]
+        x[0] = x[1]
+        x[1] = first
+
+rotateWorker(data)
+
+with open("out.json", "w") as f:
+    f.write(json.dumps(data))
```

Comment:
I think this could be `json.dump(file,data)`?

Reply:
Both versions work (if you change the parameters to `(data, file)`). But that file is just a development helper to create the resources.